frontend/script.js

```diff
@@ -0,0 +1,57 @@
+function showSideBar() {
+    const sidebar = document.querySelector('.hamburger');
+    sidebar.style.display = 'flex';
+}
+
+function hideSideBar() {
+    const sidebar = document.querySelector('.hamburger');
+    sidebar.style.display = 'none';
+}
+
+// Specify the API endpoint for user data
+const apiUrl = 'http://localhost:8080/products/';
+// Make a GET request using the Fetch API
+fetch(apiUrl)
```

Comment:
Was just thinking about the immediate involation of `fetch` here. The way I've always done it is to hook up to the window load event, and only execute my script on that event - I guess in this case the page will be ready when `fetch` is called though, unless you've noticed any timing issues?